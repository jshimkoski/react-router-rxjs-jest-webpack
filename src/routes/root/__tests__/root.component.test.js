import React from 'react';
import Root from '../Root.component';
import renderer from 'react-test-renderer';

describe('Root', () => {
  let component;
  beforeEach(() => {
    component = renderer.create(
      <Root>
        <div class="test-child-stub"/>
      </Root>
    ).toJSON();
  });

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});