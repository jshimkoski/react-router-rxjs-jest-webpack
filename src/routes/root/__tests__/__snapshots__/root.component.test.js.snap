exports[`Root renders correctly 1`] = `
<main
  className="app">
  <header>
    <h1>
      Header
    </h1>
    <nav
      className="nav">
      <img
        src="test-file-stub" />
      <ul>
        <li>
          <a
            onClick={[Function]}
            style={Object {}}>
            Home
          </a>
        </li>
        <li>
          <a
            onClick={[Function]}
            style={Object {}}>
            About
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <div
    CounterState={
      Object {
        "counter": 0,
      }
    }
    JsonPlaceholderState={
      Object {
        "postResult": Array [],
        "results": Array [],
      }
    }
    class="test-child-stub" />
</main>
`;
